### PR TITLE
feat : adding-MLENS-token

### PR DESCRIPTION
**Pull request by Gautambnsl**
# Hello MELD Team! &#128736;  
I am submitting this pull request to add details for the MELDLENS (MLENS) token in the repository. I have followed the prescribed guidelines and ensured that all necessary details are accurate and align with the required data structure. Below is an overview of the token and the associated changes.

- [x] I have checked that the format of my PR follows the contribution guidelines.

### Token Overview &#128221;
- Token Name: MELDLENS  
- Token Ticker: MLENS  
- Blockchain: MELD  
- Token Type: ERC20  
- Primary Use Case: Comprehensive Data Analytics on MELD

### Organization Details &#127970;
- Organization Name: MELDLENS  
- Website: [https://meldlens.com](https://meldlens.com)  
- Mission & Vision:  
  **MeldLens: Your All-In-One Platform for MELD Analytics**  
  MeldLens is revolutionizing how you interact with the MELD DeFi ecosystem. It’s a comprehensive analytics platform offering a suite of powerful tools to empower your decision-making and maximize your returns.

  Here’s what MeldLens offers:

  - **Profit/Loss Tracking**: Clearly visualize your gains and losses across your MELD investments.
  - **Whale Activity Monitoring**: Stay informed about large transactions that could significantly impact the market.
  - **Wallet History**: Track your complete transaction history, including deposits, withdrawals, and trades.
  - **Transaction History**: Dive deep into individual transactions with detailed information.
  - **Smart Alerts**: Set up customized alerts to be notified of critical events, such as price changes or large transactions.
  
  Additionally, users can:
  - **Earn MLENS Tokens**: Through daily airdrops and unlock exclusive features on the platform.
  - **Access Exclusive NFTs**: Stake or hold MLENS tokens to acquire unique NFTs with added benefits.

### CODE REVIEW

File: networks/meld/tokens/0xF463A50a2C51453cF6004f330aF34FB895Fc65aF.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "CMCslug": "MLENS",
```
Comment (thuyvu17) [inline networks/meld/tokens/0xF463A50a2C51453cF6004f330aF34FB895Fc65aF.json:2]: This slug does not exist on CoinMarketCap.

File: networks/meld/tokens/0xF463A50a2C51453cF6004f330aF34FB895Fc65aF.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "orgName": "MELDLENS",
+    "orgUrl": "https://meldlens.com/",
+    "tokenAddress": "0xF463A50a2C51453cF6004f330aF34FB895Fc65aF",
+    "tokenColorTheme": 1,
+    "tokenIcon": "/static/tokens/MLENS.svg",
+    "tokenName": "MELD LENS",
+    "tokenTicker": "MLENS",
+    "tokenType": "ERC20",
+    "tags": [
+    "MELDLENS"
```
Comment (thuyvu17) [inline networks/meld/tokens/0xF463A50a2C51453cF6004f330aF34FB895Fc65aF.json:14]: This tag is not currently supported. just leave it blank.

File: networks/meld/tokens/0xF463A50a2C51453cF6004f330aF34FB895Fc65aF.json (hunk 1 above)
Comment (narrasumanth) [reply networks/meld/tokens/0xF463A50a2C51453cF6004f330aF34FB895Fc65aF.json:2]: We just added liquidity, may be that is the reason? it should be available now?

Comment (rebelC0der) [reply networks/meld/tokens/0xF463A50a2C51453cF6004f330aF34FB895Fc65aF.json:2]: > We just added liquidity, may be that is the reason? it should be available now?

CMCSlug is for token, already listed on CmC. CmC does not even support the MELD chain yet, so there is no way the slug is available. Also, most of the Cardano tokens are not even listed after being active projects for years. So this should be empty and price will not be tracked for now.

Comment (rebelC0der) [reply networks/meld/tokens/0xF463A50a2C51453cF6004f330aF34FB895Fc65aF.json:2]: > We just added liquidity, may be that is the reason? it should be available now?

Just FYI, this is a token slug:
image

File: networks/meld/tokens/0xF463A50a2C51453cF6004f330aF34FB895Fc65aF.json (hunk 2 above)
Comment (Gautambnsl) [reply networks/meld/tokens/0xF463A50a2C51453cF6004f330aF34FB895Fc65aF.json]: Done

**Review by thuyvu17 — APPROVED** [review]

**Pull request merged**